src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.cs

```diff
@@ -174,7 +174,7 @@ IEnumerable<KernelCommand> commands()
 
                 yield return new SendEditableCode("language", "code");
 
-                yield return new SerializeNotebook("interactive.ipynb", new InteractiveDocument(new[]
+                yield return new SerializeInteractiveDocument("interactive.ipynb", new InteractiveDocument(new[]
```

Comment:
"Parse" and "serialize" don't go really match. Maybe rename this to something like "WriteInteractiveDocument"?

Reply:
Write / Read InteractiveDocument sounds good to me, @brettfo what is your take?

Reply:
"Parse" makes sense to me but "serialize" doesn't.